src/pages/custom-store/CustomStorePage.tsx

```diff
@@ -113,6 +117,12 @@ export default function CustomStore() {
         onSubmit={async (values) => {
           setHasSubmitError(false);
           try {
+            const signature = await signMessage({
+              message: "Confirm custom store creation"
+            });
+            if (!signature) {
+              throw new Error("Seller did not sign the custom store creation");
+            }
             const dappOrigin = getViewModeUrl(ViewMode.DAPP, BosonRoutes.Root);
```

Comment:
call verify signature to check the signer is really the seller's address (prevent from impersonation of the wallet)
(import { verifyMessage } from "ethers/lib/utils";)